_tickers.md

```diff
@@ -11,70 +11,424 @@ curl https://api.uphold.com/v0/ticker/USD
 
```

Comment:
Instead of `The above command returns the following JSON:`, it would be more accurate to say something like `The above command returns JSON in the following format:`. Otherwise there's an implicit assumption of up-to-dateness.

An alternative would be to say explicitly `As of [date], the above command returns the following JSON:`. I prefer the former option, but this would also be more correct than the current wording.